src/ucp/core/ucp_request.inl

```diff
@@ -187,3 +191,93 @@ static UCS_F_ALWAYS_INLINE void ucp_request_send_stat(ucp_request_t *req)
         UCP_EP_STAT_TAG_OP(req->send.ep, EAGER);
     }
 }
+
+static UCS_F_ALWAYS_INLINE void
+ucp_request_mrail_create(ucp_request_t *req)
+{
+    int i;
+    ucp_rndv_get_mrail_t *mrail;
+
+    ucs_trace_req("mrail create request %p", req);
+    req->send.rndv_get.mrail = (ucp_rndv_get_mrail_t *)ucs_mpool_get_inline(&(req->send.ep->worker)->mrail_mp);
```

Comment:
Too long string, extra brackets complicate reading. May be to add a macro

Reply:
no need casting here since mpool_get returns void*.